server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -1,10 +1,7 @@
 package views.admin.programs;
 
 import static com.google.common.base.Preconditions.checkNotNull;
-import static featureflags.FeatureFlag.ESRI_ADDRESS_CORRECTION_ENABLED;
-import static featureflags.FeatureFlag.INTAKE_FORM_ENABLED;
-import static featureflags.FeatureFlag.NONGATED_ELIGIBILITY_ENABLED;
-import static featureflags.FeatureFlag.PROGRAM_ELIGIBILITY_CONDITIONS_ENABLED;
+import static featureflags.FeatureFlag.*;
```

Comment:
**nitpick:** revert this change https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
This is the second time its happening. I am wondering if the bin/fmt is doing it or intelliJ

Reply:
IntelliJ does it by default, you can disable it: https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-